casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala

```diff
@@ -165,7 +171,7 @@ class HashSetCasperTest extends FlatSpec with Matchers {
       _                 <- casperEff.deploy(registerDeploy)
       createBlockResult <- casperEff.createBlock
       Created(block)    = createBlockResult
-      blockStatus       <- casperEff.addBlock(block)
+      blockStatus       <- casperEff.addBlock(block, kp2(().pure[Effect]))
```

Comment:
please replace all repetitions of this expression with an usage of a

`val ignoreDoppelgangers = kp2(().pure[Effect]) //FIXME replace with proper handler`

placed in some public object

Reply:
Also, this handler is being passed so often I'm starting to think it should be a capability required for creating a `MultiParentCasper` / for calling its methods

Reply:
I mean a typeclass over the effect type - like anything after `F` in this line:

```scala
  def sendBlockRequest[F[_]: Monad: ConnectionsCell: TransportLayer: Log: Time: ErrorHandler: RPConfAsk](...)
```

Reply:
But we need different handlers for the same instance of `MultiParentCasper`so I don't think it makes sense?

Reply:
Didn't notice that, you're right. Still, please extract the `ignoreDoppelganger` constant so that the name at least tells us what's happening.